modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -233,6 +245,34 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
     return this._network;
   }
 
+  static blockExplorer: UTXOBlockExplorer = {
+    baseUrl: "https://api.blockchair.com",
+
+    async getAddressUnspent (addressBase58: string): Bluebird<UnspentInfo[]> {
+      // make a call to blockchair apis
+      const url = `${AbstractUtxoCoin.blockExplorer.baseUrl}/${this.getFullName()}/outputs?addresses=${addressBase58}`;
+      try {
+        const res = await request.get(url);
+        if (res.hasOwnProperty('data')) {
+          const unspentInfos: UnspentInfo[] = res.data.map(function processUnspent(unspent) {
+            return {
+              address: addressBase58,
+              amount: unspent.output_total,
+              txid: unspent.hash,
+              n: unspent.index,
+            }
+          });
+          return unspentInfos;
+        }
+      } catch (e) {
+
+      }
+
+
+      // TODO throw this error after confirming the old error message
+      throw new Error(`something unexpected happened while getting unspents for address ${addressBase58}`);   // TODO make this 400 and an api error
```

Comment:
Oh, I think you just have an extraneous brace above. Because the `catch` is closed already, I would expect this to always throw.